docs/getting-started/unreal-editor/index.md

```diff
@@ -45,7 +45,9 @@ In order to package your game files, you must make sure you have [cooked](#cooki
 Once cooked, you must copy out all the files your mod replaces or adds, like maps, models, textures, etc.
 When copying you must make sure the folder structure stays the same.
 For example, a custom map at `{UE4ProjectDir}/Saved/Cooked/WindowsNoEditor/PAYDAY3/Maps/CoolMap/CoolMap.umap`
-should be copied to `{MyNewMod_P}/PAYDAY3/Maps/CoolMap/CoolMap.umap`.
+should be copied to `{MyNewMod_P}/PAYDAY3/Content/Maps/CoolMap/CoolMap.umap`.
+
+# Remember: The Mod structure needs to be identical to the Unreal Project structure
```

Comment:
This isn't the correct use of a heading. Use an admonition, something like this:

```markdown
:::info Reminder
The mod structure needs to be identical to the Unreal Project structure.
:::
```

Reply:
Thanks for the tip, I just thought it would use the same Markdown as Github so i wanted it to look like

#h1